pkg/utils/flow/flow.go

```diff
@@ -125,6 +127,7 @@ type Stats struct {
 	Succeeded TaskIDs
 	Failed    TaskIDs
 	Running   TaskIDs
+	Skipped   TaskIDs
```

Comment:
Is it necessary or is there a benefit of tracking the skipped tasks in `Stats`?

Reply:
What if we kept them in `Running`? I mean, is it necessary to have `Skipped` at all in `Stats`?

Reply:
No we can't, if we keep them in running, then it is also required to keep them in all task [ref](https://github.com/gardener/gardener/blob/a05a8670dd4bd38cd66ef91e5c5139c9c1414a25/pkg/utils/flow/flow.go#L170-L174), That means reconcile stats/progress will be reported with the skipped task also, which defeats the purpose of this PR.

Reply:
Okay, they are reported [here](https://github.com/gardener/gardener/blob/1c5fc5d5f62253ea77dba75e9c7ff3d985bffa9f/pkg/utils/flow/progress_reporter.go#L43). Thanks.